README.rst

```diff
@@ -49,6 +49,88 @@ You can run all of the tox environments using the command ``tox``.
 
 Flocker is also tested using `continuous integration`_.
 
+Cloudformation Deployment
```

Comment:
nit: Cloudformation here, but CloudFormation in all of the other files. CloudFormation looks better to me, and seems to be what Amazon uses (https://aws.amazon.com/cloudformation/)